src/components/intro/index.tsx

```diff
@@ -110,7 +110,7 @@ export default class Intro extends Component<Props, State> {
                   <div class={style.demo}>
                     <div class={style.demoImgContainer}>
                       <div class={style.demoImgAspect}>
-                        <img class={style.demoIcon} src={demo.iconUrl} alt=""/>
+                        <img class={style.demoIcon} src={demo.iconUrl} alt="" decoding="async" />
```

Comment:
@jakearchibald - I'd thought about having these images get skipped when prerendering. Right now because they're sent as static HTML, they kick of 4 requests that extend our "load" time out by quite a bit (which is more of a metrics issue, but still). Thoughts?

Reply:
The only metrics I care about are "time until useful pixels are on the screen" and "time until the user can perform the first interaction".

If one of Google's many acronyms reports the wrong time for those, it's their bug, not our's 😄 (we should report it though).